tests/main/enforcer_spec.lua

```diff
@@ -31,7 +31,7 @@ describe("Enforcer tests", function ()
         local model  = path .. "/examples/basic_model_without_spaces.conf"
         local policy  = path .. "/examples/basic_policy.csv"
 
-        local e = Enforcer:new(model, policy)
+        local e = enforcer_module:new(model, policy)
```

Comment:
@Edmond-J-A Please try not to change the module names, instead change the module import name at line 15 from `local enforcer_module = require("src.main.Enforcer")` to `local Enforcer = require("src.main.Enforcer")`.